DQM/Physics/src/CentralityDQM.cc

```diff
@@ -143,7 +143,7 @@ void CentralityDQM::analyze(const edm::Event& iEvent,
   if(cent.isValid()){
   int hibin = -999;
   if(cbin.isValid()) hibin = *cbin;
-  else edm::LogWarning("CentralityDQM") << "invalid collection: centralityBin " << std::endl;
+  //else edm::LogWarning("CentralityDQM") << "invalid collection: centralityBin " << std::endl;
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant

... BTW, these LogWarnings was just the first comment/topic from reco review, I'm not done yet.